packages/cli/src/commands/plugin/rollup.config.ts

```diff
@@ -46,6 +46,7 @@ export default {
     json(),
     typescript({
       include: `${paths.resolveTarget('src')}/**/*.{js,jsx,ts,tsx}`,
+      clean: true,
```

Comment:
This was due to an error I noticed when building in a `tmpdir`. I tracked it down to the `rollup-plugin-typescript2` caching, which I disable here. Not sure if this is a long term fix...